docs/repositories-configure/integrations/bitbucket-integration.md

```diff
@@ -52,6 +52,8 @@ Adds comments on the lines of the pull request where Codacy finds new issues. Cl
 
 ### Pull Request Summary
 
+!!! note "This feature isn't available for Bitbucket Server."
+
```

Comment:
The note isn't super visible on the page (since it's inside the "Pull Request Summary" section), but should give enough awareness about this exception if necessary.